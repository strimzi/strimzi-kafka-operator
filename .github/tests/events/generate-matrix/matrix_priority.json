{
  "event_name": "workflow_dispatch",
  "action": "workflow_dispatch",
  "inputs": {
    "pipelines": "smoke",
    "profiles": "operators,operands"
  },
  "repository": {
    "name": "ci-playground",
    "owner": { "login": "frawless" }
  }
}