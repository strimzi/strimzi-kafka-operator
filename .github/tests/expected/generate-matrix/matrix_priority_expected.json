[
  {
    "agent": "oracle-vm-2cpu-8gb-x86-64",
    "arch": "amd64",
    "pipeline": "smoke",
    "profile": "smoke",
    "timeout": 60,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 1,
    "jobName": "smoke-amd64"
  },
  {
    "agent": "oracle-vm-2cpu-8gb-arm64",
    "arch": "arm64",
    "pipeline": "smoke",
    "profile": "smoke",
    "timeout": 60,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 1,
    "jobName": "smoke-arm64"
  }
]